docs/UserGuide.adoc

```diff
@@ -555,16 +596,16 @@ Edits student with the matriculation number `A9876543K` to have:
 ====
 
 [[Sort]]
-==== Sorting a Group `sort group`
+==== Sorting a Group : `sort group`
 
-You can use this command to sort the students inside the group you want to sort
-according to the way you indicate.
+You can use this command to sort the students belonging to a specified group.
+The students will be sorted according to your specified `SORT_TYPE`.
 
-The students can be sorted alphabetically, by rating in ascending order, by rating
-in descending order or by matriculation number in ascending order.
+When a group is sorted, Student View shows all the groups of the module that the sorted group
+belongs to, as well as the students belonging to the group that has been sorted.
```

Comment:
When you sort by a specific module group, you will be switched to the Student View. Here you will see:
* the specific module highlighted
* all the groups in that module, 
* all the students in that group

smth like this